.github/workflows/lock.yml

```diff
@@ -0,0 +1,34 @@
+name: 'Lock old threads'
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
+        with:
+          issue-lock-inactive-days: '30'
+          pr-lock-inactive-days: '30'
+          issue-lock-comment: >
+            This issue has been automatically locked since there
+            has not been any recent activity after it was closed.
+            Please open a new issue for related bugs.
+          pr-lock-comment: >
+            This pull request has been automatically locked since there
+            has not been any recent activity after it was closed.
+            Please open a new issue for related bugs.
```

Comment:
I don't think there is a need to generate a notification to all the people subscribed to the issue/pr.

Reply:
Fair enough, I found the other behavior a bit too "silent", but I'm concerned about this as well.